types/whois-parsed/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for whois-parsed 1.0
+// Project: https://github.com/moneals/whois-parsed#readme
+// Definitions by: Uche Chilaka <https://github.com/uchilaka>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
DT only tests with 3.2 and above. Since this package doesn't use any 3.3+ features, you can leave this comment off.

Reply:
Made the change, but noticed CI has been failing. Looks like I was behind `master` with my fork, so trying merging in the upstream `master` branch 🤞🏾